libs/react/src/lib/link/link.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { HTMLProps } from 'react'
+
+interface LinkProps {
+  link?: string
+  text?: string
+}
+
+export const Link = ({ link, text }: LinkProps) => {
+  const props: HTMLProps<HTMLAnchorElement> = {}
+  if (link) props.href = link
+
+  return (
+    <div>
+      <a href={link} rel="noreferrer">
```

Comment:
i dont think we should define `rel="noreferrer"` for the user

Reply:
Alright, will remove it in the next commit